packages/fast-usdc/test/fast-usdc.contract.test.ts

```diff
@@ -60,12 +60,14 @@ const startContract = async (
   return { ...startKit, zoe };
 };
 
-test('start', async t => {
+// FIXME this makeTestPushInvitation forces evidence, which triggers advancing,
+// which doesn't yet work
```

Comment:
on the one hand, skipping this basic test suggests a regression.

But triggering advancing is a good problem to have. So... OK.